src/integration-test/tests/citizen/claimantResponse/data/ClaimantResponseTestData.ts

```diff
@@ -0,0 +1,50 @@
+
+interface PageSpecificValues {
+  paymentDatePage_enterDate: string
```

Comment:
I'm not sure I like the _ to break the variable name. Can we keep it camelCase for consistency? if we're going to change the convention for variables probably best to discuss first at a refinement or tech demo session

Reply:
Will change :)